### PR TITLE
Florida REST URL by GCC Args + NodeJS Florida Script 

### PR DESCRIPTION
**Pull request by diegopacheco**
@timiblossom @ipapapa 

I transformed the request into a macro parameter called FLORIDA_REQUEST so we can pass by parameter to the compiler, i did this because i have this file into a server so i can expose the URL and dont need write a proxy, it gives more flexibility to users.

Also added a simple script showing how to pass the CFLAG spaced(with is tricky in C :-) and that NodeJS Script to run Florida. Also added a config file with Florida provider that could serve as sample to other developers.

Cheers,
Diego Pacheco

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+sudo autoreconf -fvi ; sudo ./configure --enable-debug=log ; sudo make CFLAGS=-DFLORIDA_REQUEST='"\"GET /REST/v1/admin/get_seeds HTTP/1.0\r\nHost: 127.0.0.1\r\nUser-Agent: HTMLGET 1.0\r\n\r\n\""'
```
Comment (shailesh33) [inline build.sh]: This looks to be the same path as in the get_seeds.c file. I am wondering if you specify CFLAGS this way, will make ignore the CFLAGS in the MAKEFILE (which has other optimization flags etc) ?

File: src/seedsprovider/dyn_florida.c [hunk 2]
```diff
@@ -101,11 +107,11 @@ florida_get_seeds(struct context * ctx, struct mbuf *seeds_buf) {
 	uint32_t sent = 0;
 	while(sent < dn_strlen(request))
 	{
-		tmpres = send(sock, request+sent, dn_strlen(request)-sent, 0);
+		tmpres = send(sock, request + sent, dn_strlen( request )-sent, 0);
```
Comment (shailesh33) [inline src/seedsprovider/dyn_florida.c]: space around '-'

Comment (shailesh33) [inline src/seedsprovider/dyn_florida.c]: While you are at it, you might want to quickly replace the tabs with 4 spaces.

File: src/seedsprovider/dyn_florida.c [hunk 3]
```diff
@@ -17,17 +17,23 @@
...
+#endif
+
+#ifndef FLORIDA_PORT
 #define FLORIDA_PORT 8080
+#endif
 
-const char * request = "GET /REST/v1/admin/get_seeds HTTP/1.0\r\nHost: "\
-                       "127.0.0.1\r\nUser-Agent: HTMLGET 1.0\r\n\r\n";
+#ifndef FLORIDA_REQUEST
+#define FLORIDA_REQUEST "GET /REST/v1/admin/get_seeds HTTP/1.0\r\nHost: 127.0.0.1\r\nUser-Agent: HTMLGET 1.0\r\n\r\n";
```
Comment (shailesh33) [inline src/seedsprovider/dyn_florida.c]: Thank you for this!

File: build.sh (hunk 1 above)
Comment (diegopacheco) [inline build.sh]: Yes, I want to give an example how to change the PATH. For my case i dont want use that URL i want use another one. So i took some some time how to figure out the right way to pass this complex string to GCC via dynomite script that's way i keep it as SAMPLE.

File: src/seedsprovider/dyn_florida.c (hunk 2 above)
Comment (diegopacheco) [inline src/seedsprovider/dyn_florida.c]: Sure i will do.

File: src/seedsprovider/dyn_florida.c (hunk 3 above)
Comment (diegopacheco) [inline src/seedsprovider/dyn_florida.c]: ok.

**Comment by diegopacheco** [comment]
@shailesh33  DONE, did your changes.

**Comment by timiblossom** [comment]
LGTM

**Comment by timiblossom** [comment]
Can you please re-submit to Dev branch if that does take you much time?  We are still working on our branching convention and currently Dev is for new changes, not master.  Thanks

**Comment by diegopacheco** [comment]
sure @timiblossom I will do it later today.

**Comment by diegopacheco** [comment]
@timiblossom  DONE ( new PR #203) -> https://github.com/Netflix/dynomite/pull/203

**Comment by diegopacheco** [comment]
@timiblossom 

How often you guys push to master?

**Comment by timiblossom** [comment]
We will use Gitflow soon - now it is just master, dev and some ad-hoc branches

**Pull request closed without merging**